src/commonMain/kotlin/io/ktor/io/ByteOperations.kt

```diff
@@ -0,0 +1,5 @@
+package io.ktor.io
+
+public infix fun Byte.and(other: Byte): Byte = ((toInt() and 0xFF) and (other.toInt() and 0xFF)).toByte()
```

Comment:
same, why public?

Reply:
Same thoughts as for `ByteArray`, to avoid reinventing this in every library